Classes/Domain/Repository/DocumentRepository.php

```diff
@@ -789,18 +785,18 @@ public function getFirstChild($uid)
      *
      * @return int
      */
-    public function getLastChild($uid)
+    public function getLastChild(int $uid): int
     {
         $connectionPool = GeneralUtility::makeInstance(ConnectionPool::class);
         $queryBuilder = $connectionPool->getQueryBuilderForTable('tx_dlf_documents');
 
         $child = $queryBuilder
             ->select(
-                'tx_dlf_documents.uid AS uid'
+                'uid'
             )
             ->from('tx_dlf_documents')
             ->where(
-                $queryBuilder->expr()->eq('tx_dlf_documents.partof', (int) $uid)
+                $queryBuilder->expr()->eq('partof', (int) $uid)
```

Comment:
```suggestion
                $queryBuilder->expr()->eq('partof', $uid)
```
You can omit the type casting as you did in all the other methods. ;o)